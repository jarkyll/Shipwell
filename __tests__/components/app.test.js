import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from 'src/components/app';

describe('Root App', () => {
  it('should render', () => {
    const wrapper = shallow(<App />);

    expect(toJson(wrapper)).toMatchSnapshot();
  })
})