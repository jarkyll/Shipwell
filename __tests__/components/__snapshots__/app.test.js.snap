// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root App should render 1`] = `
<Fragment>
  <BrowserRouter>
    <SiteSwitch />
  </BrowserRouter>
</Fragment>
`;
